agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -25,6 +25,13 @@ using namespace son;
 #define OPERATION_SUCCESS 0
 #define OPERATION_FAIL -1
 
+/**
+ * Implementation-specific measurement period of channel utilization.
+ * Currently we use this constant value but a more elaborate solution should read it from
+ * configuration.
+ */
+static constexpr uint8_t ap_metrics_channel_utilization_measurement_period_s = 10;
```

Comment:
In this case we could use either. 
AFAIK `constexpr` is required to ensure we have a compile-time constant but here a `const` would work too. `const` is required when initialization must be done with a computation done at run time. 

I hadn't used `constexpr` before I started working for the prplMesh project. In one of my PRs I used a `#define` and @rmelotte suggested me to use `constexpr` instead. I'm using `constexpr` since then.